include/boost/beast/websocket/impl/accept.hpp

```diff
@@ -192,6 +194,14 @@ class stream<NextLayer, deflateSupported>::response_op
         , res_(beast::allocate_stable<response_type>(*this,
             sp->build_response(req, decorator, result_)))
     {
+        auto itr = req.find(http::field::expect);
+        if (itr != req.end() && itr->value() == "100-continue") // do
+        {
+            res_100_.version(res_.version());
+            res_100_.result(http::status::continue_);
+            res_100_.prepare_payload();
```

Comment:
This is ok I guess, kind of.. but.. it will be missing the Server: field no? And whatever other decorators the caller wants. Also, is the "100-continue" value case insensitive?

Reply:
I don't want to invest too much into this because I think that the number of times this code will actually execute in the wild is exactly zero.